mlkem/native/aarch64/ntt_clean.S

```diff
@@ -204,6 +214,20 @@ _ntt_asm_clean:
         load_roots_123
 
         .p2align 2
+
+        // Bounds reasoning:
+        // - There are 7 layers
+        // - When passing from layer N to layer N+1, each layer-N value
+        //   is modified through the addition/subtraction of a Montgomery
+        //   product of a twiddle of absolute value < q/2 and a layer-N value.
+        // - Recalling that |fqmul(a,t)| < q * (0.0254*C + 1/2) for
+        //   |a| < C*q and |t|<q/2, we see that the coefficients of layer N
+        //   (starting with layer 0 = input data) are bound by q * f^N(1), where
```

Comment:
What does f^N(1) mean? I can't follow that notation?

Reply:
`f(f(...(f(1))...)`

Reply:
I'll spell it out

Reply:
Lovely

Reply:
Done